config/locales/errors.en.yml

```diff
@@ -9,3 +9,23 @@ en:
       type: https://github.com/alphagov/account-api/blob/main/docs/api.md#unknown-attribute-names
       title: Unknown attribute names
       detail: Attribute names %{attributes} are unknown, have they been added to config/user_attributes.yml?
+    saved_page_not_found:
+      type: https://github.com/alphagov/account-api/blob/main/docs/api.md#saved_page_not_found
+      title: Saved page not found
+      detail: A saved page with that path cannot be found associated with that user, is the path correct?
+    saved_page_not_persisted:
+      type: https://github.com/alphagov/account-api/blob/main/docs/api.md#saved_page_not_persisted
+      title: Something has gone wrong - Adding this page failed
+      detail: This page could not be persisted to the user's account. This error has been logged.
+    saved_page_not_destroyed:
+      type: https://github.com/alphagov/account-api/blob/main/docs/api.md#saved_page_not_destroyed
+      title: Something has gone wrong - Removing this page failed
+      detail: This page could not be removed from the user's account. This error has been logged.
```

Comment:
Some of your titles start with "Something has gone wrong - ", some don't.

I think if an error is returned it's clear that something has gone wrong, so the prefix isn't needed.  Also, I've been making my titles here match the titles in the API doc.

Reply:
These are fairly vague because they're fallbacks, I can't quite imagine how folks could end up on the branch that raises this error, but I don't want to leave those routes unguarded.

The destroy I think you are right about above so I'll replace that with a 5xx error state.
But I think because we're not doing `save!` I can't use the same on the `save_page_not_persisted`.

It's vague becuase if we ever saw it we'd inevitably want to go back and write a more specific error?